mava/utils/jax_training_utils.py

```diff
@@ -106,7 +123,9 @@ def denormalize(
 
 
 def update_and_normalize_observations(
-    stats: Dict[str, Union[jnp.array, float]], observation: OLT
+    stats: Dict[str, Union[jnp.array, float]],
+    observation: OLT,
+    start_axes: int = 0,
```

Comment:
Please see the above comment about having a default slice instead of an integer.

Reply:
Or maybe it should be able to take in both an int and a list?